github-merge.py

```diff
@@ -195,7 +195,10 @@ def print_merge_details(pull, title, branch, base_branch, head_branch, acks, mes
         if acks:
             print('{}ACKs:{}'.format(ATTR_PR, ATTR_RESET))
             for name, message in acks.items():
-                print('* {} {}({}){}'.format(message, ATTR_NAME, name, ATTR_RESET))
+                if message is not None and '@' in message:
```

Comment:
Shouldn't this just print an extra line in the @ case? Right now it'll print out the warning, but you wont see the offending comment (which could be fine when containing @).

Reply:
Yes, I thought of that but began with simply doing the same thing as 6 lines down when the message contains an @.

What I'm not quite clear on is, the acks seem to already be covered by the existing '@'-check?
```python
acks = get_acks_from_comments(head_commit=head_commit, comments=comments)
message += make_acks_message(head_commit=head_commit, acks=acks)
.../...
print_merge_details(pull, title, branch, base_branch, head_branch, acks, message)
```

Reply:
Right—I'd prefer to print the comment as normal, store a flag and print a single warning at the end.

Reply:
Another possibility would be to tighten up the ACKs checking in `get_acks_from_comments`, but as a first step ensuring they are warned about seemed more prudent.

Reply:
> Right—I'd prefer to print the comment as normal, store a flag and print a single warning at the end.

Right -- will update